wagtail/admin/templates/wagtailadmin/shared/side_panels/checks.html

```diff
@@ -7,10 +7,15 @@ <h3 class="w-a11y-result__header">
             </h3>
             <div class="w-a11y-result__help" data-a11y-result-help></div>
         </div>
-        <button class="w-a11y-result__selector"
-                data-a11y-result-selector
-                aria-label="{% trans 'Show issue' %}"
-                type="button">{% icon name="crosshairs" classname="w-a11y-result__icon" %}</button>
+        <button
+            class="w-a11y-result__selector"
+            data-a11y-result-selector
+            aria-label="{% trans 'Show issue' %}"
+            type="button"
+        >
+            <span data-a11y-result-selector-text></span>
+            {% icon name="link-external" classname="w-a11y-result__icon" %}
+        </button>
```

Comment:
Tough diff, the only differences here are the icon name and addition of `<span data-a11y-result-selector-text></span>`.